### PR TITLE
Connectors list as grid

### PR DESCRIPTION
**Pull request by alejandraarri**
https://github.com/CartoDB/cartodb/issues/15208

### CODE REVIEW

File: lib/assets/javascripts/builder/components/modals/add-layer/content/imports/import-options.js [hunk 1]
```diff
@@ -61,11 +63,13 @@ module.exports = {
     view: ImportBigQueryView,
     // TODO: !config.get('bigquery_available') => omit; else !config.get('bigquery_enabled') => grayed out
     enabled: function (config, userModel) {
-      return !!config.get('bigquery_enabled');
+      // return !!config.get('bigquery_enabled');
```
Comment (jesusbotella) [inline lib/assets/javascripts/builder/components/modals/add-layer/content/imports/import-options.js]: I guess we should restore the original condition when merging to master, right?

File: lib/assets/javascripts/builder/components/modals/add-layer/content/imports/imports-selector/imports-selector-view.js [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+    // Connector is now disabled for everyone.
+    // You might be able to use the following check to determine
+    // if user/org has this connector enabled:
+    // this._userModel.get('salesforce').enabled
+    return 'disabled';
+  },
+
+  _checkMailchimpImport: function () {
+    if (!this._configModel.get('oauth_mailchimp')) {
+      // this._setImportButtonDisabled('mailchimp');
```
[the remark this answers is not in the document]
Comment (alejandraarri) [reply lib/assets/javascripts/builder/components/modals/add-layer/content/imports/imports-selector/imports-selector-view.js]: I don't know even if the Mailchimp connector works 😅

File: lib/assets/javascripts/builder/components/modals/add-layer/content/imports/imports-selector/import-catagory-section.tpl [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<div class="CDB-Text CDB-Size-medium u-tSpace-xl u-bSpace--xl">
```
[the remark this answers is not in the document]
Comment (alejandraarri) [reply lib/assets/javascripts/builder/components/modals/add-layer/content/imports/imports-selector/import-catagory-section.tpl]: 🙈

File: lib/assets/javascripts/builder/components/modals/add-layer/content/imports/import-options.js (hunk 1 above)
Comment (alejandraarri) [reply lib/assets/javascripts/builder/components/modals/add-layer/content/imports/import-options.js]: Actually, we shouldn't. But this is temporary as I'm going to develop the DB/BQ connectors UI in next iterations.

File: lib/assets/javascripts/builder/components/modals/add-layer/content/imports/imports-selector/imports-selector-view.js [hunk 4]
```diff
@@ -0,0 +1,287 @@
...
+      return 'disabled';
+    }
+
+    if (!this._userModel.featureEnabled('mailchimp_import')) {
+      return 'disabled';
+    }
+    return 'enabled';
+  },
+
+  _checkPostgreSQLImport: function () {
```
[the remark this answers is not in the document]
Comment (alejandraarri) [reply lib/assets/javascripts/builder/components/modals/add-layer/content/imports/imports-selector/imports-selector-view.js]: Yes, I like the idea. But let me implement this in the following iteration as I need to refactor this code.

File: lib/assets/javascripts/builder/components/modals/add-layer/content/imports/imports-selector/imports-selector-view.js [hunk 5]
```diff
@@ -0,0 +1,287 @@
...
+ *  - Create a check function here if needed, if not will appear
+ *    always enabled (for everybody!).
+ *
+ */
+
+module.exports = CoreView.extend({
+  className: 'ImportOptions',
+
+  events: {
+    'click .js-goNext': '_moveToNextTabs',
```
[the remark this answers is not in the document]
Comment (alejandraarri) [reply lib/assets/javascripts/builder/components/modals/add-layer/content/imports/imports-selector/imports-selector-view.js]: No, but I think this clean up is developed in another branch 😅

**Comment by csubira** [comment]
Acceptance done following the steps in description -> 🆗 👌 👌 👌

**Pull request merged**